.github/workflows/build-terraform-cli.yml

```diff
@@ -60,15 +58,14 @@ jobs:
           arch: ${{ inputs.goarch }}
           reproducible: nope
           instructions: |-
-            mkdir dist out
-            set -x
-            go build -ldflags "${{ inputs.ld-flags }}" -o dist/ .
-            zip -r -j out/${{ env.ARTIFACT_BASENAME }} dist/
-      - uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
-        with:
-          name: ${{ env.ARTIFACT_BASENAME }}
-          path: out/${{ env.ARTIFACT_BASENAME }}
-          if-no-files-found: error
+            go build -ldflags "${{ inputs.ld-flags }}" -o "$BIN_PATH" -trimpath -buildvcs=false
+            cp LICENSE "$TARGET_DIR/LICENSE.txt"
+      - name: Copy license file to config_dir
+        if: ${{ matrix.goos == 'linux' }}
+        env:
+          LICENSE_DIR: ".release/linux/package/usr/share/doc/${{ inputs.package-name }}"
+        run: |
+          mkdir -p "$LICENSE_DIR" && cp LICENSE "$LICENSE_DIR/LICENSE.txt"
```

Comment:
For Debian-based distributions in particular there's a convention for this directory to contain a file named `copyright` which follows [a machine-readable format](https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/).

While of course we aren't compelled to follow that convention since we aren't intending these packages to go into the main Debian archive, following the distribution conventions might make it easier for folks to use existing tools to catalog the licenses of software they have installed, to aid with verifying license compliance.

As far as I know the specified format for these files is a relatively new thing and there are various existing official Debian packages that instead just place the license text verbatim into the file without any special formatting, and so a pragmatic compromise might be to just change the name of the file from `LICENSE.txt` to `copyright` but leave its contents otherwise unchanged, rather than trying to maintain or generate a separate file that uses the Debian-specific file format. :thinking: 

(I expect there's probably an established convention for RPM-based distributions too, but I'm not familiar with that ecosystem.)

Reply:
FYI I've flagged this for the RelEng team to consider and they'll be looking into this suggestion this week.

Reply:
RelEng has decided they'd like to stick with the original (current) approach, keeping the licenses as .txt files.